client/cli/src/params/network_params.rs

```diff
@@ -145,6 +145,10 @@ pub struct NetworkParams {
 		verbatim_doc_comment
 	)]
 	pub sync: SyncMode,
+
+	/// Disable persisting peers between the node restarts.
+	#[clap(long)]
+	pub disable_peers_persistence: bool,
```

Comment:
Nitpick: personally `disable_peer_persistence` feels a little bit more natural to me, but that might be just me; feel free to ignore